lib/tile-bag.rb

```diff
@@ -0,0 +1,69 @@
+# require_relative '../lib/player'
+# require_relative '../lib/scoring'
+
+require 'pry'
+
+module Scrabble
+  class TileBag
+
+    attr_accessor :tiles_remaining, :tile_bag
+
+    def initialize
+      @tiles_remaining = 98
+
+      @tile_bag = {
```

Comment:
This is **not** a great structure for tracking the tiles.  This is because doing it this way makes it equally likely any letter will be drawn despite the fact that there are 9 "A" tiles and only 2 "B" tiles.